shopify/order/add_free_product_to_new-customer_first_order/mesa.json

```diff
@@ -0,0 +1,99 @@
+{
+  "key": "add_free_product_to_new-customer_first_order",
```

Comment:
3. In the key, it looks like you missed changing a dash - to an underscore. Please change to "add_free_product_to_new_customer_first_order".